.github/workflows/rainforestqa.yml

```
@@ -103,3 +103,14 @@ jobs:
           RAINFOREST_API_TOKEN: ${{ secrets.RAINFORESTQA_TOKEN }}
           # The run group for tests to run during CI, now has start URL built-in
           RUN_GROUP: ${{ env.RUN_GROUP }}
+
+      - name: Slack Notification
```

If the previous step fails, this won't be reached
```suggestion
      - name: Slack Notification
        if: failure()
```


https://stackoverflow.com/a/58859404/288906